src/hooks/useEvm/useEvm.ts

```diff
@@ -0,0 +1,20 @@
+import { CHAIN_NAMESPACE, fromChainId } from '@shapeshiftoss/caip'
+import { useMemo } from 'react'
+import { getChainAdapters } from 'context/PluginProvider/PluginProvider'
+import { selectFeatureFlags } from 'state/slices/preferencesSlice/selectors'
+import { useAppSelector } from 'state/store'
+
+export const useEvm = () => {
```

Comment:
Is this ever going to do something else than return `supportedEvmChainIds` ?

Reply:
I suspected that it would, but I don't know what yet.